spec/features/course/assessment/submission/worksheet_spec.rb

```diff
@@ -78,6 +78,28 @@
         end
       end
 
+      scenario 'I can edit my existing comment', js: true do
```

Comment:
I'm for reducing test time, but I'm not sure about this from the software engineering perspective. @allenwq any opinions?